source/fuzz/fuzzer_pass_apply_id_synonyms.cpp

```diff
@@ -110,13 +107,35 @@ void FuzzerPassApplyIdSynonyms::Apply() {
         // the synonym's |index_size| is 0), or an id that will hold the result
         // of extracting a synonym from a composite (when the synonym's
         // |index_size| is > 0).
-        uint32_t id_with_which_to_replace_use;
-        if (synonym_to_try->index_size() == 0) {
-          id_with_which_to_replace_use = synonym_to_try->object();
-        } else {
+        uint32_t id_with_which_to_replace_use = synonym_to_try->object();
+
+        // If synonym represents some element of a composite structure, inserts
+        // a composite extract instruction before the proper instruction.
+        if (synonym_to_try->index_size() > 0) {
           id_with_which_to_replace_use = GetFuzzerContext()->GetFreshId();
+          auto instructionToInsertBefore = use_inst;
```

Comment:
Do I initialize it to `nullptr`? Because it is a pointer to an instruction object.

Reply:
Done.

Reply:
Either is fine, but I have a slight preference for not initializing it.  The reason is that both branches should initialize it.  If you forget in one of the branches, a pedantic compiler will complain because there are paths where it remains uninitialized.  If you initialize it to nullptr then it is initialized along every path, so if you accidentally forget to assign to it the compiler will not complain.  See what I mean?

Reply:
Yes, I understood.